test/protractor/specs/spec.js

```diff
@@ -0,0 +1,20 @@
+describe('Okta Sign-In Widget', function() {
+  var username = element(by.name('username'));
```

Comment:
Hmmm, does this work, and will it work across multiple tests? If it's a protractor convention, it might be okay, but it feels confusing to write this like this - i.e. where we're initializing our username object before we've navigated to the page.

It'd be clearer to do something like:
```javascript
// Use a function where it's clear that we're invoking it at the right time
function getUserName() {
  return element(by.name('username'));
}
```